src/components/skillsLayout/style.ts

```diff
@@ -1,8 +1,15 @@
 import styled from "styled-components";
 
-export default styled("div")`
+interface SDiv {
+  showSmall?: boolean;
+}
+
+export default styled("div")<SDiv>`
   display: grid;
-  grid-template-columns: repeat(4, 1fr);
+  grid-template-columns: repeat(
+    ${(props) => (props.showSmall ? "2" : "4")},
+    1fr
```

Comment:
according to what I see on screenshot, and in code above. U drilling props, creating new element in swith case, changing interfaces, and in the end all changes what u make it grid-template-column. Meanwhile all what u needed its writing here media-query for max-width. I dont insist on changes, but I totally disagree with approach like this.

Reply:
ohh, realy. I did not even think about it